adafruit_minimqtt.py

```diff
@@ -126,8 +125,8 @@ def __init__(self, socket, broker, port=None, username=None,
             self.client_id = client_id
         else:
             # assign a unique client_id
-            self.client_id = 'cpy{0}{1}'.format(microcontroller.cpu.uid[randint(0, 15)],
-                                                randint(0, 9))
+            self.client_id = 'cpy{0}{1}'.format(randint(0, int(time.monotonic()*100)%1000),
```

Comment:
@jerryneedell 
> According to MQTT V3. 1 Protocol Specification, "The Client Identifier (Client ID) is between 1 and 23 characters long

I do not think we will ever generate a CID >23 char since this step is performed during initialization in use-cases where a CircuitPython device performs a reset or at boot-up.

However, what about if a long `time.monotonic` value is returned after a delayed initialization or delayed WiFi connection (if the power in the house is down, the AP will take a while to reboot, code will retry until the modem and AP is fully online). 

Do you feel we should truncate a generated `client_id` here to 23chars, before it fails the check below and raises a `ValueError`

Reply:
this code will never use more than 3 digits from the time.monotonic()  so the whole thing will only be ```cpyxxxyy```  so 7 digits at most.. Do you want it longer. It was even fewer the way it was oringally done.

Reply:
Nope, 1<=cid<=23, so it's valid. I'll mergein and release! Thanks for fixing this.